### PR TITLE
Supporting Fedora Builds

### PR DESCRIPTION
**Pull request by aodhan-domhnaill**
Build is failing because it is using outdated libdigidoccpp dev package

Pending completion of https://github.com/open-eid/libdigidocpp/pull/410

### CODE REVIEW

**Review by metsma — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -22,7 +22,30 @@ set_env( MOBILEID_URL "https://dd-mid.ria.ee/mid-api" CACHE STRING "URL for Mobi
...
+set(CPACK_PACKAGE_NAME "qdigidoc4")
+set(CPACK_PACKAGE_RELEASE 1)
+set(CPACK_PACKAGE_CONTACT "Andrus Randveer")
+set(CPACK_PACKAGE_VENDOR "RIA")
+set(CPACK_PACKAGING_INSTALL_PREFIX "/usr/")
+set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION
+        "/usr/local" "/usr/local/etc" "/usr/share/man"
+        "/usr/share/man/man1" "/usr/lib64/pkgconfig" "/usr/share/icons/hicolor"
+        "/usr/share/applications" "/usr/share/icons" "/usr/share/mime" "/usr/share/mime/packages"
+        "/usr/share/icons/hicolor/16x16" "/usr/share/icons/hicolor/16x16/apps" "/usr/share/icons/hicolor/16x16/mimetypes"
```
Comment (aidan-plenert-macdonald) [inline CMakeLists.txt:36]: This seems like an anti-pattern. I'm not sure how to avoid it. Without this, there are a bunch of install conflicts.

**Comment by aidan-plenert-macdonald** [comment]
I was able to successfully sign a document using the libdigidocpp I built from https://github.com/open-eid/libdigidocpp/pull/410 and this build of qdigidoc4

File: CMakeLists.txt (hunk 1 above)
Comment (metsma) [reply CMakeLists.txt:36]: Im not sure what is installed under /usr/local.

**Comment by metsma** [comment]
Can you fix the conflict

**Pull request closed without merging**